includes/class-sensei-quiz.php

```diff
@@ -1091,6 +1091,116 @@ public function get_user_assessment_feedback( $lesson_id , $user_id = 0 ){
 	} // end get_user_assessment_feedback
 
      /**
+      * Save the user's assessment feedback
+      *
+      * For this function you must supply all three parameters. If will return false one is left out.
+      * The data will be saved on the lesson ID supplied.
+      *
+      * @since 1.7.5
+      * @access public
+      *
+      * @param string $assessment_feedback
+      * @param int $lesson_id
+      * @param int $user_id
+      *
+      * @return false or int $feedback_saved
```

Comment:
Sorry picky mode activated. 

format for multiple returns in docblocks is to use |. so for example this should be:

@return bool|int